wordle-web/src/App.vue

```diff
@@ -1,86 +1,13 @@
 <template>
   <header>
-    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
-
     <div class="wrapper">
-      <HelloWorld msg="You did it!" />
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">Class</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
+      <NavBar class="ma-10"></NavBar>
```

Comment:
This menu is kinda disorienting and confusing to interact with, could use refinement.

Reply:
I'm not sure what about the menu is confusing. If you have more specific recommendations we would be happy to change it though.